mne/time_frequency/tfr.py

```diff
@@ -724,7 +724,10 @@ def tfr_array_morlet(epoch_data, sfreq, freqs, n_cycles=7.0,
                      n_jobs=1, verbose=None):
     """Compute time-frequency transform using Morlet wavelets.
 
-    Convolves epoch data with selected Morlet wavelets.
+    Same computation as the function tfr_morlet.
+    Difference: tfr_morlet takes epochs in the mne.Epochs format as input.
+    This function, tfr_array_morlet, takes epochs in the numpy.array format
+    as input.
```

Comment:
```suggestion
    Same computation as `~mne.time_frequency.tfr_morlet`, but operates on
    :class:`NumPy arrays <numpy.ndarray>` instead of `~mne.Epochs` objects.
```